src/components/table/table.tsx

```diff
@@ -0,0 +1,45 @@
+import { Component, h, Prop, Element } from '@stencil/core';
+import TabulatorTables from 'tabulator-tables';
+
+export interface Column {
+    title: string;
+    field: string;
+}
+@Component({
+    tag: 'limel-table',
+    styleUrl: 'table.scss',
+    shadow: true,
+})
+export class Table {
+    /**
+     * table data to be displayed
+     */
+    @Prop()
+    public tableData: any[] = [];
+    /**
+     * object for column definitions
+     * {
+     *    title -required
+     *    field- required
+     * }
```

Comment:
I think we should remove these. I think absolutely we should document this in some way but this is not the right place. We have not written it like this for any other components, e.g. `ListItem` in the list component